### ISSUE TITLE
Urban Construction Metaphor is a Different Situation

### ISSUE THREAD

**Issue by sajkaj** [w3c/wcag#939]
Detlev Fischer (IE) on October 14commented on
[https://w3c.github.io/wcag/conformance-challenges/#introduction](Introduction:)
   When applying the urban construction metaphor to a large website, the situation seems very different: you work in a development system, trial, acceptance-test (incl. accessibility) and then release changes - so the
   physical metaphor seems somewhat misleading.
||                                   ||
   See above - I think this is not a valid argument. You cannot prototype and test a new building or road without disrupting the implemented reality, but you can (and do all the time) in an IT einvironment.

**Comment by sajkaj** [comment]
Urban infrastructure analogy has been removed in the [Working Draft](https://github.com/w3c/wcag/tree/conformance-challenges-working-draft/conformance-challenges).

**Comment by slauriat** [comment]
Challenge #2 still has a heading of 'Challenge #2: Large, complex, and dynamic websites are always “under construction”'

**Comment by detlevhfischer** [comment]
I see nothing wrong with that in principle as long it does not reference bricks’n mortar — but maybe it is better to say: ”are continuously being modified”

**Comment by sajkaj** [comment]
I reopened 941 this morning because that seemed to me a better articulated version of this issue. I also marked it for FPWD. So, we’ll add text to explain how users might be impacted by constantly changing content, even though no one intentionally exposes users to under construction content.

Note that the urban metaphor has been removed from the document for some time now, e.g. it’s not in the Editor’s Draft.

An in depth status update is about to post on list.

Janina
